cmd/corald/main.go

```diff
@@ -48,7 +48,7 @@ func main() {
 		writeTimeout = 30 * time.Second
 	)
 
-	if err := app.Run(":4001", routes.API(), readTimeout, writeTimeout); err != nil {
+	if err := app.Run(":3000", routes.API(), readTimeout, writeTimeout); err != nil {
```

Comment:
It actually does under the hood. ardanlabs/kit already takes the PORT variable from the environment and uses it and only defers to this variable when it's not in the environment.